articles/extensions/delegated-admin.md

```diff
@@ -0,0 +1,163 @@
+---
+description: The Delegated Administration extension allows you to expose the Users dashboard to a group of users, without allowing them access to the dashboard.
+---
+
+# Delegated Administration
+
+The **Delegated Administration** extension allows you to expose the [Users dashboard](${uiURL}/#/users) to a group of users, without having to provide access to them to the [dashboard](${uiURL}/#/). Instead the [Users dashboard](${uiURL}/#/users) is exposed as an Auth0 client. Let's see how this is done.
+
+**NOTE**: This extension is currently available only for the public cloud. Extensions are not yet supported in the [appliance](/appliance).
+
+## Configure the extension
+
+Let's start with creating a new client application. Navigate to [Clients](${uiURL}/#/applications) and click on the **+Create Client** button. Set a name (we will name ours *Users Dashboard*) and choose *Single Page Web Applications* as client type. Click on **Create**.
+
+![](/media/articles/extensions/delegated-admin/create-client.png)
+
+Navigate to *Settings > Show Advanced Settings > OAuth* and set the **JsonWebToken Signature Algorithm** to *RS256*. Click **Save Changes**.
+
+![](/media/articles/extensions/delegated-admin/set-rs256.png)
+
+While on the *Settings* tab set the **Allowed Callback URLs**. This varies based on your location.
```

Comment:
@sandrinodimattia At the moment there is not variable I can use so I listed the options. I registered [this issue](https://github.com/auth0/auth0-docs/issues/994). When resolved I will update the doc.